DQM/Integration/python/test/online_customizations_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+def customise(process):
+    
+    process.options = cms.untracked.PSet(
```

Comment:
Instead of replacing the entire ```options``` block and possibly removing other items in it I suggests something like

```
     if not hasattr(process, "options"):
        process.options = cms.untracked.PSet()

     process.options.numberOfThreads = cms.untracked.uint32(1)
     process.options.numberOfStreams = cms.untracked.uint32(1)
```